main/res/values/strings.xml

```diff
@@ -1025,6 +1029,8 @@
     <string name="waypoint_duplicate">Duplicate waypoint</string>
     <string name="waypoint_duplicated">Waypoint duplicated</string>
     <string name="waypoint_copy_of">Copy of</string>
+    <string name="waypointcalc_notes">Notes:</string>
```

Comment:
You should probably use `Notes: %s` so that other languages are more free in how they represent things. This is something that we need to do for the rest of cgeo.